src/server/zset_family.cc

```diff
@@ -2370,6 +2432,71 @@ void ZSetFamily::ZPopMinMax(CmdArgList args, bool reverse, ConnectionContext* cn
   OutputScoredArrayResult(result, range_params, cntx);
 }
 
+void ZSetFamily::GeoAdd(CmdArgList args, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 0);
+
+  // TODO: to handle options and multiple elements
+  ZParams zparams;
+
+  string_view longitude = ArgS(args, 1);
```

Comment:
You could return an error (saying it's unsupported) if `longitude` is one of the strings `NX`, `XX`, `CH`

Reply:
see above: 
it's not a command going to prod tomorrow, it's a and we check the validity later for everything. the important thing - it should not crash anything.